benchmarks/python/test_cat.py

```diff
@@ -0,0 +1,487 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+import pytest
+import torch
+from nvfuser import FusionDefinition, DataType
+from .core import run_benchmark, with_executor
+
+
+# Qwen2 fusion that involves concatenation. Note that there are no Mistral-Nemo
+# benchmarks in this file, because they were all equivalent to this fusion. The
+# fusion below appears repeatedly in both networks.
+#
+# The numbers are arbitrary; this was the 11th fusion in the forward pass.
+@pytest.mark.parametrize("ex", ["thunder", "torchcompile"])
```

Comment:
```Suggestion
@pytest.mark.parametrize("executor", ["thunder", "torchcompile"])
```

We use the parameter name to skip all but nvfuser (manual) by default. For any other benchmark we use `--benchmark-{executor}` flag. This is to limit the CI resources needed for our nightly/weekly runs.
Based on when these benchmarks should be run, the appropriate flags will be added to the CI script. CC: @xwang233

Reply:
Ahh, didn't realize the exact name was relevant. Applied, thanks!
I think this came up multiple times though, i'll update the other places too